<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>867</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>openGL</string>
  </property>
  <property name="windowIcon">
   <iconset resource="opengl.qrc">
    <normaloff>:/img/openGL.svg</normaloff>:/img/openGL.svg</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>50</width>
    <height>50</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="opengl" name="openGLWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actStart"/>
   <addaction name="actStop"/>
   <addaction name="actReset"/>
  </widget>
  <action name="actStart">
   <property name="text">
    <string>开始模拟</string>
   </property>
  </action>
  <action name="actStop">
   <property name="text">
    <string>暂停模拟</string>
   </property>
  </action>
  <action name="actReset">
   <property name="text">
    <string>重置模拟</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>opengl</class>
   <extends>QOpenGLWidget</extends>
   <header>opengl.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="opengl.qrc"/>
 </resources>
 <connections/>
</ui>
